docs/_admin/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Project Administration
+order: -1
+---
+
+The web-platform-tests project is more than just tests! This section documents
+all the information necessary to administer the infrastructure which makes the
+project possible.
+
+## Secrets
+
+Some aspects of the infrastructure are only accessible to administrators. These
+include access credentials for third-party services (e.g. continuous
+integration providers), data used to prove identity (e.g. SSL certificates),
+and deployment information for the project's running systems (e.g.
+[w3c-test.org](https://w3c-test.org)).
+
+Project                   | Secret                   | Owners
+--------------------------|--------------------------|-------------
+[results-collection]      | root SSH keys            | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[results-collection]      | Password for app secrets | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[pull-requests-dashboard] | root SSH keys            | ???
+[pull-requests-dashboard] | Password for app secrets | boaz@bocoup.com, geoffers@gmail.com, jgraham@hoppipolla.co.uk, lukebjerring@google.com, mike@w3.org
+[web-platform-tests]      | TravisCI authorization   | ???
+
+Note that SSL certificates for all HTTPS-enabled domains are retrieved via
+[Let's Encrypt](https://letsencrypt.org/), so that data does not represent an
+explicitly-managed secret.
+
+## Third-party account owners
```

Comment:
Sauce?

Reply:
@zcorpan asked the same thing. Tho account we use for the results-collection are present in encrypted form in that repository, so they are available to anyone who has the "Password for app secrets" documented above. Because of this, and because re-using them in other contexts could cause interference, I think they should be considered an implementation detail of that project and omitted from this organization-wide document.